i18n/en.json

```diff
@@ -108,9 +108,9 @@
 	"srf-paramdesc-renderer": "Select a graph/chart renderer",
 	"srf-paramdesc-filling": "Individual filling option",
 	"srf-paramdesc-theme": "Select a grid theme",
-	"srf-paramdesc-pagelength": "define datatable page length",
-	"srf-paramdesc-lengthmenu": "define datatable length menu",
-	"srf-paramdesc-datatables-columnstype": "define datatable columns type",
+	"srf-paramdesc-pagelength": "Specifies the length of the page for the datatable",
+	"srf-paramdesc-lengthmenu": "Specifies the length of the menu for the datatable",
+	"srf-paramdesc-datatables-columnstype": "Specifies the type for the columns of the datatable",
```

Comment:
Isn't the "for the datatable" part already clear from context? And should it not be "of the datatable" in all cases?

`length of the page` can be shortened to `page length`

Reply:
Sure, why not.